shiny-ci-cd/actions/ci-cd-renv.yml

```diff
@@ -8,17 +8,32 @@ name: CI-CD
 
 jobs:
   CI-CD:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.config.os }}
+
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+
+    strategy:
+      # we keep a matrix for convenience, but we would always want to run on one
+      # single OS and R version
+      matrix:
+        config:
+          - {os: ubuntu-20.04, r: 'release'}
+
     env:
       RENV_PATHS_ROOT: ~/.local/share/renv
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
shouldn't we add a comment here about this being the Personalized Access Token necessary to connect to GitHub?

Reply:
Resolved as belongs to same topic of adding more comments to these files in general.